Octokit.Tests/Clients/BlobClientTests.cs

```diff
@@ -122,7 +122,7 @@ public void CanDeserializeBlob()
             Assert.NotNull(response.Body);
             Assert.Equal(blobResponseJson, (string)response.HttpResponse.Body);
             Assert.Equal(100, response.Body.Size);
-            Assert.Equal(EncodingType.Utf8, response.Body.Encoding);
+            Assert.Equal(EncodingType.Utf8, response.Body.Encoding.ParsedValue);
```

Comment:
I quickly wrote the convention tests to assert this and there are 42 enums that have at least 1 member that doesnt have the `[Parameter()]` attribute specified.

Im not averse to fixing them ALL up as part of this PR, since this PR is effectively "redoing" how we do enums.